custom/samveg/case_importer/validators.py

```diff
@@ -107,6 +110,43 @@ def run(cls, row_num, raw_row, fields_to_update):
         return fields_to_update, error_messages
 
 
+class UploadLimitValidator(BaseValidator):
+    @classmethod
+    def run(cls, row_num, raw_row, fields_to_update, import_context):
+        error_messages = []
+        owner_name = raw_row.get(OWNER_NAME)
+        call_value, call_number = cls._get_latest_call_value_and_number(raw_row)
+        if owner_name and call_number:
+            if cls._upload_limit_reached(import_context, owner_name, call_number):
```

Comment:
Should the row, or the call columns, be skipped if the limit is reached?

Reply:
the row should be skipped entirely, that is the requirement. 

It's just to have some sort of randomization, to have cases for different owners in HQ.

Reply:
I thought to skip the row, you needed to only return `error_messages`. This looks like it appends to `error_messages` but then returns the tuple.

Reply:
to skip the row eventually in core HQ there needs to be at least one error message from the validators/operations.
The return for the function here, is expected to be the final set of fields to be updated, if they have been changed and the error messages.

I see that the return for fields to be updated is causing a confusion because we are not modifying it anywhere, till now.
I was thinking more from a generic perspective of extensions, that the extensions can modify the fields to be updated and hence it should return that back.
I do have a requirement of adding a new case property to the case, so I would be using it there eventually.